app/src/app/api/v0/emissions-factor/route.ts

```diff
@@ -2,33 +2,58 @@ import { db } from "@/models";
 import { apiHandler } from "@/util/api";
 import createHttpError from "http-errors";
 import { NextRequest, NextResponse } from "next/server";
-import uniqBy from "lodash/uniqBy";
 import { Op } from "sequelize";
+import { fetchEmissionsFactorRequest } from "@/util/validation";
 
-export const GET = apiHandler(async (req: NextRequest, _context: {}) => {
-  const { searchParams } = new URL(req.url);
-  const inventoryId = searchParams.get("inventoryId");
-  const referenceNumber = searchParams.get("referenceNumber");
-  const methodologyId = searchParams.get("methodologyId");
-
-  const city = await db.models.City.findOne({
-    attributes: ["regionLocode"],
-    include: [
-      {
-        model: db.models.Inventory,
-        as: "inventories",
-        attributes: [],
-        where: {
-          inventoryId: inventoryId,
+const filterMappings: Record<string, any> = {
+  "fuel-type-wood/wood-waste": "fuel-type-wood-wood-waste",
```

Comment:
This should probably be fixed in the data (seeder or MI JSON depending on where this comes from), rather than "painted over" in JS...

Reply:
This is a good point. I think we can delay this till when we are updating the manual input json file. then we can replace the keys across board